spec/support/comment.cr

```diff
@@ -3,6 +3,6 @@ require "./post"
 class Comment < LuckyRecord::Model
   table comments do
     field body : String
-    belongs_to post : Post
+    belongs_to post : Post?
```

Comment:
I think this should be changed back because a comment should *always* have a post. Might be worth adding a new table for `Employee` that belongs to `Manager`:

```crystal
class Employee < LuckyRecord::Model
  table :employees do
    belongs_to manager : Manager?
  end
end
```

Reply:
There is a spec/associations_spec.cr where you could test this out. One for testing when the associated record is nil, and another for when it is present

Reply:
Good idea. I created the tables and specs and found an error when the parent model is nil. The `find(id)` method throws `invalid input syntax for integer: ""` so I added an overload `find(id : Nil)` that throws a `RecordNotFoundError` instead.

Is this okay, or should there be different behavior for this case?